index.md

```diff
@@ -3,15 +3,17 @@ layout: default
 title:  JuliaOpt&#58; Optimization packages for the Julia language
 ---
 
-# 4th annual JuMP-dev
+# New webpage
+
+The new webpage for the JuMP ecosystem is: [https://jump.dev](https://jump.dev)
```

Comment:
Could we word this as, "We are transitioning to a new webpage for the JuMP ecosystem: [https://jump.dev](https://jump.dev)."

Reply:
sure! done